### DISCUSSION TITLE
Transformers in PostCss plugin?

### DISCUSSION THREAD

**Discussion by brizjin** [unocss/unocss#3475] [Q&A]
Is it possible to use transformers with PostCss plugin? Can not to set up them work( In my project I use webpack with postcss, but can not make work not webpack directly not by postcss. In uno webpack plugin I somehow can not make webpack virtual module import `import 'uno.css'` react does not see it. Not sure I should move to Vite to make things work) And there is no any full examples for webpack or postcss througt webpack(

**Comment by userquin** [comment]
maybe you can check https://github.com/unocss/unocss/tree/main/examples/vite-vue3-postcss

**Reply by brizjin** [reply]
There is no usage of any transformers at this example. I trying to make work [Compile class transformer](https://unocss.dev/transformers/compile-class), with no luck so far) Also I did tried webpack plugin, and also can not make it work, virtual import 'uno.css' does not work(
